addon/components/horizontal-bar-chart.js

```diff
@@ -244,19 +246,26 @@ export default ChartComponent.extend(FloatingTooltipMixin,
       },
       y: this.get('barThickness') / 2,
       dy: '.35em',
-      'text-anchor': (d) => d.value < 0 ? 'end' : 'start',
+      'text-anchor': (d) => {
+        if ((d.value < 0) || ((d.value === 0) && this.get('hasNegativeValues') && !this.get('hasPositiveValues'))) {
```

Comment:
I was considering something like that, but I wanted to use a general function for both "value" and "grouping" labels. That would require the directions to switch depending on the calling context. Couldn't find a particularly clean way of doing it